datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad

```diff
@@ -134,45 +134,45 @@
 - electric_vehicle_profile_5_share = 0.0
 - solar_pv_profile_1_share = 1.0
 - solar_pv_profile_2_share = 0.0
-- annual_infrastructure_cost_gas = 8.236905182997704
+- annual_infrastructure_cost_gas = 8.236905182997703e-06
 - areable_land = 14.61
-- co2_emission_1990 = 0.340686003887848
+- co2_emission_1990 = 0.346136633
 - coast_line = 0.0
 - interconnector_capacity = 60000.0
 - total_land_area = 41.21
-- number_of_buildings = 7009.0
-- number_of_cars = 38091.0
-- number_of_residences = 43045.0
-- number_of_inhabitants = 88602.0
-- offshore_suitable_for_wind = 258.769864280363
+- number_of_buildings = 1693.0
+- number_of_cars = 37955.0
+- number_of_residences = 42035.0
+- number_of_inhabitants = 90837.0
+- offshore_suitable_for_wind = 30060.0
```

Comment:
Another big change. Amstelveen does not have large lakes nor seas, right?

Reply:
No :-) But this number has been set the same for all municipalities